.github/workflows/ci.yaml

```diff
@@ -0,0 +1,45 @@
+name: Debos Continuous Integration
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Build Debos
+    runs-on: ubuntu-latest
```

Comment:
can we run on debian buster instead?

Reply:
unfortunately not: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#github-hosted-runners

Reply:
hmmm in that case and given we're already running the docker container build in a (seperate?) job that builds on buster can we use this job to build and test with the latest go instead?

Then we have two actually different test when it comes to compilations and unit tests etc at least

Reply:
running these tests allow you to get annotations which show against the source code, like: https://github.com/obbardc/debos/commit/4e04945f42c2623c3a9c783b92aaf2f5ef2ac975

unfortunately github actions isn't clever enough to create annotations from the docker run: https://github.com/obbardc/debos/commit/79291ca1d8f6ce0f0288aa8ae84bb9ae3ab6c062

if you don;t think that annotations are useful then we can just drop this whole stage and just run the docker tests

Reply:
yep, agree i will make the build run on latest go.